docs/how-to-use-customized-configuration.md

```diff
@@ -1,294 +0,0 @@
-          ...
-          scalar.dl.ledger.proof.enabled=true
-          scalar.dl.ledger.auditor.enabled=true
-          scalar.dl.ledger.proof.private_key_path=/keys/private-key
-      ```
-    * ScalarDL Auditor
-      ```yaml
-      auditor:
-        auditorProperties: |
-          ...
-          scalar.dl.auditor.cert_path=/keys/certificate
-          scalar.dl.auditor.private_key_path=/keys/private-key
-      ```
-
-In this example, you need to mount a **private-key** and a **certificate** file under the `/keys` directory in the container. And, you need to mount files named `private-key` and `certificate`. You can use `extraVolumes` and `extraVolumeMounts` to mount these files.
-
-1. Set `extraVolumes` and `extraVolumeMounts` in the custom values file using the same syntax of Kubernetes manifest. You need to specify the directory name to the key `mountPath`.
-   * Example
-        * ScalarDL Ledger
-          ```yaml
-          ledger:
-            extraVolumes:
-              - name: ledger-keys
-                secret:
-                  secretName: ledger-keys
-            extraVolumeMounts:
-              - name: ledger-keys
-                mountPath: /keys
-                readOnly: true
-          ```
-       * ScalarDL Auditor
-         ```yaml
-         auditor:
-            extraVolumes:
-              - name: auditor-keys
-                secret:
-                  secretName: auditor-keys
-            extraVolumeMounts:
-              - name: auditor-keys
-                mountPath: /keys
-                readOnly: true
-          ```
-
-1. Create a `Secret` resource that includes key and certificate files.  
-   You need to specify the file name as keys of `Secret`.
-   * Example
-       * ScalarDL Ledger
-         ```console
-         kubectl create secret generic ledger-key-secret \
-           --from-file=private-key=./ledger-key.pem
-         ```
-       * ScalarDL Auditor
-         ```console
-         kubectl create secret generic auditor-key-secret \
-           --from-file=private-key=./auditor-key.pem \
-           --from-file=certificate=./auditor-cert.pem
-         ```
-
-1. Deploy Scalar products with the above custom values file.  
-   After deploying Scalar products, key and certificate files are mounted under the `/keys` directory as follows.
-   * Example
-       * ScalarDL Ledger
-         ```console
-         $ ls -l /keys/
-         total 0
-         lrwxrwxrwx 1 root root 18 Jun 27 03:12 private-key -> ..data/private-key
-         ```
-       * ScalarDL Auditor
-         ```console
-         $ ls -l /keys/
-         total 0
-         lrwxrwxrwx 1 root root 18 Jun 27 03:16 certificate -> ..data/certificate
-         lrwxrwxrwx 1 root root 18 Jun 27 03:16 private-key -> ..data/private-key
-         ```
-
-   Please refer to the [Getting Started with Scalar Helm Charts](./getting-started-scalar-helm-charts.md) for more details on how to use each Helm Chart.
-
-## Set flags with the ScalarDL Schema Loader chart
```

Comment:
I moved this section to [Flags configurations (Optional based on your environment)](https://github.com/scalar-labs/helm-charts/blob/main/docs/configure-custom-values-scalardl-schema-loader.md#flags-configurations-optional-based-on-your-environment) in the [Configure a custom values file for ScalarDL Schema Loader](https://github.com/scalar-labs/helm-charts/blob/main/docs/configure-custom-values-scalardl-schema-loader.md).